rimage/man_kbl.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2017, Intel Corporation.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms and conditions of the GNU General Public License,
+ * version 2, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ */
+
+#include "css.h"
+#include "cse.h"
+#include "plat_auth.h"
+#include "manifest.h"
+#include <config.h>
+#include <version.h>
+
+#define KBL_HARDWARE_BUFFER_BASE	0xBE500000
+#define KBL_HARDWARE_BUFFER_LEN		0x4A000
+
```

Comment:
this is common to multiple platforms so should be v1_5 instead of KBL

Reply:
the file name as well should not be man_kbl but something like man_intel_1_5

Reply:
We are now using man_apl.c man_cnl.c, I think there should have difference in 1.8 for these two different platform. and I think we still can have man_apl.c using 1.8 and 1.5 manifest. Will kbl still using 1.8 manifest in principle? @lgirdwood Do you have comments on this?

Reply:
Will the hardware buffer length and base address changed according to different version of manifest or different platform? Or the hardware buffer length and base address will keep the same for all the platform?

Reply:
@plbossart this might be different on the other platforms, so probably best to keep them as is and put a comment to this effect. 
@zhuyingjiang can you put a TODO comment here stating that these values also need checked on other v1.5 platforms and aligned if all identical.

Reply:
OK, regarding the hardware buffer length, I will keep and comment on it. For man_intel_1_5, I will first add man_skl.c and add FIXME in it.